### DISCUSSION TITLE
Cannot establish connection with remote device

### DISCUSSION THREAD

**Discussion by 7u5a** [TannerGabriel/WebRTC-Video-Broadcast#72] [General]
I can get the example to run at my localhost, but trying to connect from another pc on the network is not possible for me.
Am I doing something wrong or shouldn't it be possible?

Best regards Tue.

**Comment by TannerGabriel** [comment]
If you are on the same network (no firewall between the two computers), it should work out of the box. If you still don't get a WebRTC connection, I recommend adding a [TURN server](https://github.com/TannerGabriel/WebRTC-Video-Broadcast#adding-a-turn-server).

**Reply by 7u5a** [reply]
Microsoft Defender, would that be a problem?
